app/enrichment/schemas/header.js

```diff
@@ -3,7 +3,7 @@ const Joi = require('joi').extend(require('@joi/date'))
 module.exports = Joi.object({
   sourceSystem: Joi.string().required(),
   schemeId: Joi.number().integer().required(),
-  ledger: Joi.string().valid('AP', 'AR'),
+  ledger: Joi.string().valid('AP', 'AR').required(),
```

Comment:
Can these constant values be defined and imported from a module as they are used throughout the service?

- Ledgers (already exists: `app/ledgers.js`)
- Currency
- Schedule
- Date format

This is quite a substantial change and involves a lot of test changes too so perhaps is best suited as a follow-on ticket in itself?

Reply:
For ledger, currency and schedule, I've made the changes in this PR.  I agree it makes sense and as they're just strings and we have existing tests covering them, I think the risk is low so happy to do it now.

For the date format, I was in two minds about whether it makes it easier or more complicated to understand.  I've changed it to use a constant and I _think_ it's better.